Components/PalindomeChecker.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
+
+export const PalindomeChecker = () => {
+  const [input, setInput] = useState('');
+  const [isPalindrome, setIsPalindrome] = useState(false);
+
+  function isArrayLengthEven (arrLength: number) {
+    if (arrLength % 2 === 0){
+      return true;
+    }
+      return false;
+  }
+
+  const isItAPalindrome = (): void => {
+      let arrToCheck = input.split('');
+      let numOfChecks;
+
+      const isArrayEven = isArrayLengthEven(arrToCheck.length);
+
+      if (isArrayEven) {
+        numOfChecks = arrToCheck.length/2;
+      } else (
+        // There's got to be a smarter way to do this that reads easier
+        numOfChecks = (((arrToCheck.length - 1) / 2) + 1)
```

Comment:
would this work?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/ceil

Reply:
Ooooh - Math.ceil is perf!